runtime/doc/freedesktop.txt

```diff
@@ -0,0 +1,46 @@
+*freedesktop.txt* For Vim version 9.1. Last change: 2024 Mar 20
+
+
+		  VIM REFERENCE MANUAL    by Bram Moolenaar
+
+
+				      *freedesktop* *X-desktop-group* *xdg*
+This file contains some information on what it is freedesktop and the XDG
+families of specifications.
+
+Freedesktop is a project aiming to develop specifications to enable
+interoperability between Unix-like operating systems.
+
+Vim tries to conform, for some extent, to the specifications made by
+freedesktop (see |xdg-base-dir|).
+
+==============================================================================
+XDG Base Directory Specification	*xdg-base-dir* *$XDG_CONFIG_HOME*
+
+The XDG Base Directory Specification aim to define standard locations for the
+multilple data e configuration files used by applications. In order to divide
```

Comment:
Typo

Reply:
I fixed those when merging. You should probably compare against what was committed instead.

Reply:
@chrisbra https://github.com/vim/vim/pull/14559